lib/node_modules/@stdlib/array/bool/benchmark/benchmark.reduce.js

```diff
@@ -0,0 +1,67 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var isBoolean = require( '@stdlib/assert/is-boolean' ).isPrimitive;
+var Boolean = require( '@stdlib/boolean/ctor' );
+var pkg = require( './../package.json' ).name;
+var BooleanArray = require( './../lib' );
+
+
+// FUNCTIONS //
+
+/**
+* Reducer function.
+* @private
+* @param {any} acc - accumulated value
+* @param {boolean} value - current array element
+* @param {integer} index - current array index
+* @returns {any} accumulated value
+*/
+function reducer( acc, value ) {
+	return Boolean( acc & value );
+}
```

Comment:
Not sure whether it is good way for benchmarking.

Reply:
I'd use the "sum" example from the README. Also, make sure you understand the difference between `&` and `&&`. They are not the same thing.